### PR TITLE
feat: change address generation rule

### PR DESCRIPTION
**Pull request by shaojunda**

### CODE REVIEW

File: app/utils/ckb_utils.rb [hunk 1]
```diff
@@ -38,24 +38,40 @@ def self.generate_lock_script_from_cellbase(cellbase)
...
 
+  def self.full_payload_address(lock_script)
+    code_hash = lock_script.code_hash
+    args = lock_script.args
+    format_type = lock_script.hash_type == "data" ? "0x02" : "0x04"
+    first_arg = args.first
+
+    return unless args.all? { |arg| CKB::Utils.valid_hex_string?(arg) }
+
+    CKB::Address.new(first_arg).generate_full_payload_address(format_type, code_hash, args)
```
Comment (classicalliu) [inline app/utils/ckb_utils.rb]: It seems `first_arg` is useless, is this an SDK implement problem?

Comment (shaojunda) [reply app/utils/ckb_utils.rb]: SDK default params is blake160

Comment (shaojunda) [reply app/utils/ckb_utils.rb]: I changed SDK

Comment (shaojunda) [reply app/utils/ckb_utils.rb]: thanks.

**Pull request merged**